packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -125,10 +125,12 @@ After finishing building the bundle, you can also try the sample
 
 1. Replace the `ResourceConnectionString` in the `samples/Server/appsettings.json` file with the connection string from your Azure Communication Services resource, which is listed in the Azure Portal under _Keys_.
 
-2. Run this command to start the server:
+2. Replace the `EndpointUrl` in the `samples/Server/appsettings.json` file with the connection string and remove "endpoint=" from the beginning of the string.
```

Comment:
This isn't quite right. They should get the endpoint URL directly from the ACS resource.

The hack before this PR was parsing the connection string as a URL and then using just the $HOST part of it, which ignored other information in the connection string.

Reply:
I don't quite understand what you mean. LGTM

Reply:
Users should paste the endpoint directly from the portal instead of extracting it out of their connection string:
![image](https://user-images.githubusercontent.com/2684369/149821761-c93a52e7-c99e-43dd-9ff2-d97397462a51.png)
(^red box)

Reply:
Update the doc to get value from ACS Resource.